src/main.rs

```diff
@@ -56,10 +57,33 @@ pub struct WorkerParameters {
   /// # Provider
   /// Name of the provider used for the transcription
   provider: String,
+  /// # Output Format
+  /// Output format between EBU-TT-D and json
+  output_format: Option<String>,
   destination_path: String,
   source_path: String,
 }
 
+#[derive(Debug, Clone, Deserialize, JsonSchema)]
+enum OutputFormat {
+  EbuTtD,
+  Json,
+}
+
+impl FromStr for OutputFormat {
+
+    type Err = mcai_worker_sdk::MessageError;
+
+    fn from_str(input: &str) -> Result<OutputFormat> {
+        match input {
+            "EBU_TT_D"  => Ok(OutputFormat::EbuTtD),
+            "JSON"  => Ok(OutputFormat::Json),
+            _      => Ok(OutputFormat::EbuTtD),
```

Comment:
Maybe you should log a warning message, to notify the specified format is not supported, don't you think?